src/middlewared/middlewared/plugins/disk.py

```diff
@@ -1061,6 +1101,61 @@ def label_to_dev(self, label, geom_scan=True):
 
         await self.sync(dev)
 
+    @private
+    def format(self, disk, swapgb):
+
+        job = self.middleware.call_sync('disk.wipe', disk, 'QUICK')
+        job.wait_sync()
+        if job.error:
+            raise CallError(f'Failed to wipe disk {disk}: {job.error}')
+
+        # Calculate swap size.
+        swapsize = swapgb * 1024 * 1024 * 2
+        # Round up to nearest whole integral multiple of 128 and subtract by 34
```

Comment:
Comment does not match code: where is 34 and what is 34?

Reply:
This was also copied from original code, I guess its just outdated comment, i will remove